### PR TITLE
ACE interactions for adding door tape numbers & closeable doors

### PR DESCRIPTION
**Pull request by sevn10**
**When merged this pull request will:**
- Add ability via ACE interaction on the outside of the helo to choose or clear the use of door tape numbers
- Add ability via ACE interaction on the outside of the helo to close and open cabin doors

### CODE REVIEW

**Comment by BroBeansCPG** [comment]
Do checks need to be added to determine whether or not something is blocking the doors? 

Consider https://github.com/Project-Hatchet/public_h-60/blob/9fd7416d1b8d54a6ad8bfc9caf0f5f73407fbeeb/addons/uh60_misc/functions/fnc_canInteractCabinDoor.sqf

**Comment by BroBeansCPG** [comment]
Maybe explore the using of ace interaction on the door handle memory point

**Comment by sevn10** [comment]
That could be good, but I believe I'll need to be reminded on how to find that memory point. I can ping you on discord to clarify.

**Comment by A26mike** [comment]
What is the current status of this.  @sevn10 ?

@AdozenBABYdogs give this a test and let me know the results

**Comment by sevn10** [comment]
@A26mike I need to talk to you or brobeans, I'm not sure how to find the ACE memory point. I could try to revise this afterwards.

**Comment by BroBeansCPG** [comment]
> What is the current status of this. @sevn10 ?
> 
> @AdozenBABYdogs give this a test and let me know the results

Also the door numbers were added in #130

**Comment by BroBeansCPG** [comment]
> @A26mike I need to talk to you or brobeans, I'm not sure how to find the ACE memory point. I could try to revise this afterwards.

DM me on discord

**Comment by AdozenBABYdogs** [comment]
> **When merged this pull request will:**
> 
> * Add ability via ACE interaction on the outside of the helo to choose or clear the use of door tape numbers
> * Add ability via ACE interaction on the outside of the helo to close and open cabin doors

Door numbers work, doors open from the outside

**Comment by AdozenBABYdogs** [comment]
Suggested next step

- add animation for exiting with doors closed
- add check for if doors are open, if they are not, animate the player getting out
- disable player's from being able to fire while doors are closed

**Comment by YannikH** [comment]
@sevn10 The mempoint names for the door handles are cabindoor_L_handle and cabindoor_R_handle

**Comment by ampersand38** [comment]
Please see vtx_uh60_misc_fnc_canInteractCabinDoor for [un]locking the door seats on MH and HH models.

How often do you need to change door numbers mid-mission? It's probably be better left to mission-maker setup, I'm thinking. Maybe separate PRs for doors and numbers.

File: addons/uh60_misc/XEH_postInit.sqf [hunk 1]
```diff
@@ -8,3 +8,186 @@ private _action = [
...
+private _displayName = "Cabin Doors";
+private _door = "";
+private _desired = "";
+private _condition = {true};
+private _statement = {};
+private _action = ["vtx_cabin_doors",_displayName, "", _statement, _condition] call ace_interact_menu_fnc_createAction;
+["vtx_H60_base", 0, ["ACE_MainActions"], _action, true] call ace_interact_menu_fnc_addActionToClass;
+
+_displayName = "Close Left Door";
+_condition = {((_this select 0) animationPhase "cabindoor_l") == 0};
```
Comment (ampersand38) [inline addons/uh60_misc/XEH_postInit.sqf:172]: Needs to [un]lock door seats. See vtx_uh60_misc_fnc_canInteractCabinDoor

**Review by ampersand38 — CHANGES_REQUESTED** [review]

**Comment by Riverman02** [comment]
These features have been added, can't this be closed?

**Pull request closed without merging**